Overwrite install section for FreeBSD

pawelgrzeda opened the pull request.

pawelgrzeda committed:
Merge pull request #1 from travis-ci/default

update default branch

pawelgrzeda committed:
Update .travis.yml

michal-at-travisci commented on .travis.yml:
```diff
@@ -11,6 +11,12 @@ jobs:
+      - popd
+      - sudo python -m pip install virtualenv==20.0.2
       script:
         - __dots() { while true ; do echo -en . ; sleep 1 ; done } ; __dots &
```
That's after @DamianSzymanski experiments - can u remove lines 20 - 22 (script section) here?

michal-at-travisci committed:
clean up for FreeBSD install section

Removing all dots, which were put temporarily to suppress `after_failure:  cat /tmp/python-build.*.log` output - log was so large, that it resulted in job terminated due to too large job.

In future: instead of cat -> upload that log somewhere to grab it later and analyze

michal-at-travisci reviewed (approved):
Looks good for me, requires follow-up tests
